ush/forecast_predet.sh

```diff
@@ -54,6 +54,20 @@ common_predet(){
   CDATE=${CDATE:-2017032500}
   ENSMEM=${ENSMEM:-000}
 
+  if [[ "${DOIAU}" = "YES" ]]; then
```

Comment:
Line 138 should also be moved above line 57 so that the value of `DOIAU` is not reset in `FV3_predet`.
I have a question about the variables `IAUFHRS`, `IAU_DELTHRS` and `IAU_OFFSET`.  Are they still FV3 specific?  I cannot imagine `IAU_OFFSET` to be FV3 specific as it is used to calculate the model start time when IAU is ON|OFF.